README.md

```diff
@@ -28,13 +28,17 @@ Features include:
 
 ### Current Status
 
-This library has matured quite a bit, but should still be considered incomplete. 
+> [!TIP]
+> MSRV Note: This crate is now on Rust `1.70.0`.
+> If you want to use the newest version of `exrs` with an older Rust version, you can still do that, by forcing Rust to use a an older version of the `half` crate via `cargo update -p half --precise 2.2.1`.  `half 2.3.0` and higher have an MSRV above 1.61.
```

Comment:
The suggested workaround is different from what is being tested in the CI, but it's a tad silly because `-Zminimal-versions` requires _nightly Rust_ (on the topic of supporting older MSRV)...

That only really works if users are using `Cargo.lock` and okay with using a _nightly_ toolchain to generate that (maybe on a different machine or CI if they're locked to a specific version at build-time only). `cargo update --precise ...` is a bit more approachable, and could be used in the `exrs` CI instead?

Reply:
I would be fine with just suggesting the existence of `-Zminimal-versions`, even if using that temporarily requires a nightly cargo version. I would assume that people struggling with an older compiler know these special approaches.

As you seem to know about this topic, I'd trust your input on that.

On a different note, I would ask you to put this whole section into the readme section where the installation command is shown. Ok?

Reply:
Let's showcase both commands in the readme, it's a `[!TIP]` block anyway.

Moved under `### Usage`!